liquidity/liquidity.go

```diff
@@ -25,6 +30,9 @@ type Config struct {
 	// LoopOutRestrictions returns the restrictions that the server applies
 	// to loop out swaps.
 	LoopOutRestrictions func(ctx context.Context) (*Restrictions, error)
+
+	// LND provides us with access to lnd's main rpc.
+	LND lndclient.LightningClient
```

Comment:
This is pretty aggressive. Is that that annoying linter rule again?

Reply:
Meaning `LND` vs `Lnd`, no linter problems, just conventional go?

Reply:
We already have for example `LndServices`